iroha/src/queue.rs

```diff
@@ -113,7 +115,7 @@ impl<W: WorldTrait> Handler<GetPendingTransactions> for Queue<W> {
     type Result = PendingTransactions;
     async fn handle(
         &mut self,
-        GetPendingTransactions: GetPendingTransactions,
+        _get_pending_transactions: GetPendingTransactions,
```

Comment:
It was intentional. Previously there was deconstruction of zero size structure. It should break after addition of fields to it. So it is better to have it as it was before

Reply:
I think we need a comment about that there ;)

Reply:
reverted

Reply:
> I think we need a comment about that there ;)
This is kinda silly to comment some language constructs